core/src/test/java/org/apache/accumulo/core/util/GuavaCacheCounterUtil.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Preconditions;
+import com.google.common.cache.Cache;
+
+/**
+ * Unit test helper for counting guava cache entries. Guava cache size() is approximate, and can
+ * include garbage-collected entries, so we iterate to get the actual cache size
+ */
+public class GuavaCacheCounterUtil {
+  private static final Logger LOG = LoggerFactory.getLogger(GuavaCacheCounterUtil.class);
+  private static final int reties = 5;
```

Comment:
```suggestion
  private static final int retries = 5;
```

Reply:
defining it as a variable was for clarity and avoid a "magic number" at least this way it seemed clearer what it is doing.

Reply:
Spelling fixed in c3e02e336c